### PR TITLE
[DOCS] function worker docs enhancement

### PR DESCRIPTION
**Pull request by abhilashmandaliya**
* updated function worker startup instructions
* updated a note to clarify on which admin-url should be used when functions-worker is run separately
* changed the order of 'Setup: Configure Functions runtime' and 'Setup: Pulsar Functions Worker' as the worker page's first line was pointing to the runtime page which is bad UX

### CODE REVIEW

**Review by freeznet — APPROVED** [review]
@abhilashmandaliya thanks, LGTM.

**Comment by abhilashmandaliya** [comment]
> @abhilashmandaliya thanks, LGTM.

@freeznet this will be carried forward to all the coming versions right? I made changes to 2.8.0 only

**Comment by freeznet** [comment]
@abhilashmandaliya i believe you have to change `site2/docs` contents as well, need to confirm with @Anonymitaet

File: site2/website/versioned_docs/version-2.8.0/functions-worker.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ This section illustrates how to run `functions-worker` as a separate process in
 ![assets/functions-worker-separated.png](assets/functions-worker-separated.png)
 
 > Note    
-> In this mode, make sure `functionsWorkerEnabled` is set to `false`, so you won't start `functions-worker` with brokers by mistake.
+> In this mode, make sure `functionsWorkerEnabled` is set to `false`, so you won't start `functions-worker` with brokers by mistake. Also, while accessing the `functions-worker` to manage any of the functions, the `pulsar-admin` CLI tool or any of the client should use the `workerHostname` and `workerPort` that you set in [Worker parameters](#worker-parameters) to generate an `--admin-url`.
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.8.0/functions-worker.md:69]:
```suggestion
> In this mode, make sure `functionsWorkerEnabled` is set to `false`, so you won't start `functions-worker` with brokers by mistake. Also, while accessing the `functions-worker` to manage any of the functions, the `pulsar-admin` CLI tool or any of the clients should use the `workerHostname` and `workerPort` that you set in [Worker parameters](#worker-parameters) to generate an `--admin-url`.  
```

File: site2/website/versioned_sidebars/version-2.8.0-sidebars.json [hunk 2]
```diff
@@ -25,8 +25,8 @@
     ],
     "Pulsar Functions": [
       "version-2.8.0-functions-overview",
-      "version-2.8.0-functions-worker",
       "version-2.8.0-functions-runtime",
+      "version-2.8.0-functions-worker",
```
Comment (Anonymitaet) [inline site2/website/versioned_sidebars/version-2.8.0-sidebars.json:29]: why change the sequence?

**Comment by Anonymitaet** [comment]
> @abhilashmandaliya i believe you have to change `site2/docs` contents as well to make it forward to coming versions, need to confirm with @Anonymitaet

Yes, please help make the same changes to `master` (site2/docs) and `2.8.1` as well, thanks.

**Comment by abhilashmandaliya** [comment]
@Anonymitaet 
changed the order of 'Setup: Configure Functions runtime' and 'Setup: Pulsar Functions Worker' as the worker page's first line was pointing to the runtime page which is bad UX

**Comment by abhilashmandaliya** [comment]
> > @abhilashmandaliya i believe you have to change `site2/docs` contents as well to make it forward to coming versions, need to confirm with @Anonymitaet
> 
> Yes, please help make the same changes to `master` (site2/docs) and `2.8.1` as well, thanks.

this is done @Anonymitaet

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**